otomoto/settings.py

```diff
@@ -10,7 +10,9 @@
 #     https://doc.scrapy.org/en/latest/topics/spider-middleware.html
 
 BOT_NAME = 'otomoto'
-
+LOG_FILE = "otomoto.log"
+LOG_ENABLED = True
+LOG_LEVEL = 'INFO'
```

Comment:
I would leave to the user if he wants to log.  I am skeptical right now about this change.

Reply:
OK